configs/typescript.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint'
+  ],
+  rules: {
```

Comment:
how about using the recommended rules as a base

```
  "extends": [
//    "eslint:recommended", this can be omitted since it is already included in configs/base
    "plugin:@typescript-eslint/eslint-recommended",
    "plugin:@typescript-eslint/recommended"
  ]
```

from https://www.npmjs.com/package/@typescript-eslint/eslint-plugin

Reply:
The recommendation rule includes @ typescript-eslint / no-explicit-any, which means that we cannot use the `any` type, we must set specific types for all variables, which will bring more workload.

Reply:
Ok, I think that's a valid concern, and I am fine with us using `any` at least for the conversion process. But I do think we should still use the `recommended`s, and then just override them in `rules` something like `'@typescript-eslint/no-explicit-any': 'warn'`

Reply:
Override is ok, I agree with you

Reply:
@mattrabe can you check again? just pushed a commit